packages/module/patternfly-docs/content/extensions/virtual-assistant/design-guidelines.md

```diff
@@ -0,0 +1,165 @@
+---
+section: PatternFly-AI
+subsection: Chatbot
+id: Overview
+sortValue: 1
+source: design-guidelines
+---
+
+import "./images.css"
+
+## Elements 
+
+<div class="ws-docs-content-img">
+![Main elements of a ChatBot.](./img/chatbot-elements.svg)
+</div>
+
+1. **Container:** The window that contains the entire ChatBot experience and all of its components.
+1. **Header:** A persistent region at the top of the ChatBot window that contains navigation, branding, and actions.
+1. **Navigation:** A menu that contains navigational options, including access to the conversation history.
+1. **Messages:** Elements of the conversation between a ChatBot and user. For more details, refer to the [ChatBot message guidelines](#messages).
+1. **Attachments:** Details about files that a user has uploaded to the ChatBot.
+1. **Footer:** A persistent region at the bottom of the ChatBot window that contains the message bar and the footnote.
+1. **Footnote (optional):** A persistent, short message that contains any legal disclaimers or important information about the ChatBot. Footnotes are optional, but strongly recommended. For more information, refer to the [ChatBot footnote guidelines](#footnotes).
+1. **Toggle:** The button that allows users to open and close the ChatBot window. When the ChatBot is opened, the toggle should appear below the ChatBot window.
```

Comment:
Hmm that's a fair point. My only thought with "launch toggle" is that the toggle also handles minimizing, rather than just launching (ik this is nitpicky). We could call it the "ChatBot toggle" to make it more standalone? I think the slight repetitiveness of "ChatBot" isn't distracting in this situation?

Reply:
I agree @edonehoo Let's call it ChatBot Toggle